miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/SampleSingleCellDto.java

```diff
@@ -0,0 +1,35 @@
+package uk.ac.bbsrc.tgac.miso.dto;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+
+import uk.ac.bbsrc.tgac.miso.core.data.SampleSingleCell;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonTypeName(value = SampleSingleCell.SAMPLE_CLASS_NAME)
+public class SampleSingleCellDto extends SampleTissueProcessingDto implements SampleSingleCellRelative {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This class has 7 parents which is greater than 5 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AMaximumInheritanceDepth)